kubemarine/procedures/check_iaas.py

```diff
@@ -276,6 +277,40 @@ def system_distributive(cluster):
         tc.success(results=", ".join(detected_supported_os))
 
 
+def check_access_to_thirdparties(cluster: KubernetesCluster):
+    detect_preinstalled_python(cluster)
+    broken = []
+
+    # Load script for checking sources
+    all_group = cluster.nodes['all']
+    local_path = utils.get_resource_absolute_path("resources/scripts/check_thirdparty_avaliability.py",
+                                                  script_relative=True)
+    random_temp_path = "/tmp/%s.py" % uuid.uuid4().hex
+    all_group.put(local_path, random_temp_path, sudo=True, binary=False)
+
+    for destination, config in cluster.inventory['services'].get('thirdparties', {}).items():
+        # Check if curl
+        if config['source'][:4] != 'http' or '://' not in config['source'][4:8]:
+            continue
+        # Check with script
+        common_group = cluster.create_group_from_groups_nodes_names(config.get('groups', []), config.get('nodes', []))
+        for node in common_group.get_ordered_members_list(provide_node_configs=True):
+            python_executable = cluster.context['nodes'][node['connect_to']]['python']['executable']
+            res = node['connection'].sudo("%s %s %s" % (python_executable, random_temp_path, config['source']), warn=True)
```

Comment:
I think that work with thirdparties should be same when checking and installing. Let's imagine, for example, that it will require to install thirdparty with `curl -k` option. It will be challenging to remember all different ways to work with the thirdparty URL. This is also a good chance to get rid of dependency on curl, so probably we can make more complex python script which will able to both install and check the thirdparty?

Reply:
Firstly, kubemarine can't install thirdparty with -k option now (options are hardcoded).
Secondly, during this task we check only availability, not real installation, only for this reason code in python will be different for that two cases, if we decide to implement those changes.
Finally,  curl is required not only for thirdparties, e.g. in paas_check tasks. For this reason if we want to get rid of curl, we should do it complexly in separate PR...
Also do we really need this?

Reply:
As I checked, redirect works for urllib  
For this reason we can use python for this task and curl for others cases